doc/dev/cephadm/developing-cephadm.rst

```diff
@@ -228,15 +228,15 @@ When completed, you'll see::
 
 Then you can reload your Dashboard browser tab.
 
-Cephadm DiD (Docker in Docker) box development environment
-==========================================================
+Cephadm box container (Podman inside Podman) development environment
+====================================================================
 
 As kcli has a long startup time, we created an alternative which is faster using
-Docker inside Docker. This approach has its downsides too as we have to
+Podman inside Podman. This approach has its downsides too as we have to
 simulate the creation of osds and addition of devices with loopback devices.
 
-Cephadm's DiD environment is a command which requires little to setup. The setup
-requires you to get the required docker images for what we call boxes and ceph.
+Cephadm's box environment is a command which requires little to setup. The setup
+requires you to get the required podman images for what we call boxes and ceph.
```

Comment:
Consistent capitalization please. Ceph should be capitalized, probably Docker and Podman.  I also think these sentences don't add much.

Perhaps:
Cephadm's box environment is simple to set up.

Reply:
Add a space "to set up" and reverse the next sentence to be less ambiguous:

Cephadm's box environment is simple to set up. The setup requires you to
get the required Podman images for Ceph and what we call boxes.

Reply:
Done.